### ISSUE TITLE
Problem on Lollipop when connecting to InAppBillingService

### ISSUE THREAD

**Issue by radzio** [serso/android-checkout#31]
I see something like this in my Crashlytics

```
java.lang.RuntimeException: Unable to create application com.scrobblefm.App: java.lang.IllegalArgumentException: Service Intent must be explicit: Intent { act=com.android.vending.billing.InAppBillingService.BIND }

Caused by: java.lang.IllegalArgumentException: Service Intent must be explicit: Intent { act=com.android.vending.billing.InAppBillingService.BIND }
       at android.app.ContextImpl.validateServiceIntent(ContextImpl.java:1701)
       at android.app.ContextImpl.bindServiceCommon(ContextImpl.java:1800)
       at android.app.ContextImpl.bindService(ContextImpl.java:1778)
       at android.content.ContextWrapper.bindService(ContextWrapper.java:538)
       at org.solovyev.android.checkout.Billing$DefaultServiceConnector.connect(Billing.java:938)
       at org.solovyev.android.checkout.Billing.connectOnMainThread(Billing.java:269)
       at org.solovyev.android.checkout.Billing.access$400(Billing.java:51)
       at org.solovyev.android.checkout.Billing$3.run(Billing.java:261)
       at org.solovyev.android.checkout.MainThread.execute(MainThread.java:57)
       at org.solovyev.android.checkout.Billing.connect(Billing.java:258)
```

**Comment by serso** [comment]
I also get crash reports with the same stack trace but I don't understand why this happens. I fixed the problem for Android 5 devices by setting the explicit package name in the intent. It seems that the devices where the crash occurs have custom Android version or strange environment - the user names are f.e. jenkins or android-build. So may be these are not real people but some kind of automation tool running.
How many crash reports did you get?

**Comment by radzio** [comment]
I've got something like 5 errors. But all of them were reported from real devices which  is really strange.

**Comment by serso** [comment]
OK, I'll fix the crash by catching the exception but billing on those devices won't work.

**Comment by serso** [comment]
It's fixed, please update to version 0.7.2. It seems that devices without Google Play Services were ones affected (I'm not 100% though).

**Comment by radzio** [comment]
Thanks a lot!
